graders/python/python_autograder/pl_result.py

```diff
@@ -13,20 +13,8 @@ class PLTestResult(unittest.TestResult):
     """
 
     error_message = (
-        "The grading code failed -- sorry about that!\n\n"
-        "This may be an issue with your code.\nIf so, you can "
-        "take a look at the traceback below to help debug.\n"
-        "If you believe this is an issue with the grading code,\n"
-        "please notify the course staff.\n\n"
-        "The error traceback is below:\n"
-    )
-
-    major_error_message = (
-        "The grading code was not able to run.\n"
-        "Please notify the course staff "
-        "and include this entire message,\n"
-        "including the traceback below.\n\n"
-        "The error traceback is:\n"
+        "Thre was an error while grading your code.\n\n"
+        "Look at the traceback below to help debug your code:\n"
```

Comment:
I'd suggest add a note encouraging students to "review the question text to ensure your code matches the expected requirements, such as variable names, function names and parameters" or something in this regard.

Reply:
Yeah, I like language like that. Softens it a little, even if they don't know how to read the error. "Incorrect syntax or not matching what the grader is expecting might cause these errors. Review the traceback below to help debug your code." Could add a "Use the Report an Issue with this question button on the right if you think the grader is malfunctioning"

Reply:
Added in da7c1105428e8f0a91b4e27395af67615a83e9fd. I opted to not include the "Report an issue" text because I fear that it will cause the same problem as we currently have with the proctors, but for course staff 😄